packages/@uppy/robodog/package.json

```diff
@@ -31,6 +31,7 @@
     "@uppy/core": "file:../core",
     "@uppy/dashboard": "file:../dashboard",
     "@uppy/dropbox": "file:../dropbox",
+    "@uppy/box": "file:../box",
```

Comment:
excluding box from global module for now as it will initially be released as beta
```suggestion
```